Error: Can't resolve 'papaparse/papaparse.min.js' wiht ngx-papaparse@6

**Describe the bug**

Today I upgraded ngx-papaparse to version 6.0 and then my build failed with error message: 

```
./node_modules/ngx-papaparse/fesm2020/ngx-papaparse.mjs:3:0-50 - Error: Module not found: Error: Can't resolve 'papaparse/papaparse.min.js' in '/path/to/project/node_modules/ngx-papaparse/fesm2020'

```
Papaparse was not added as dependency in package.json because it was not necessary with ngx-paparse@5.

Is it now necessary to add papaparse as dependency to use ngx-papaparse@6 ?

--- a/ng-package.json
+++ b/ng-package.json
@@ -7,5 +7,8 @@
   "lib": {
     "entryFile": "src/public_api.ts"
   },
-  "deleteDestPath": true
+  "deleteDestPath": true,
+  "allowedNonPeerDependencies": [
+    "papaparse"
+  ]
 }

--- a/package.json
+++ b/package.json
@@ -43,9 +43,7 @@
     "e2e": "ng e2e"
   },
   "dependencies": {
-    "tslib": "^2.0.0"
-  },
-  "peerDependencies": {
+    "tslib": "^2.0.0",
     "papaparse": "^5.3.2"
   },
   "devDependencies": {

